adopterAPI.json

```diff
@@ -201,22 +220,41 @@
           "ApplicationsEndpoints"
         ],
         "summary": "Gets all applications. Requires any role. For shelter, applications are filtered only for that shelter. For adopter, applications are filtered only for that adopter. Admin gets them all.",
+        "parameters": [
+          {
+            "name": "pageNumber",
+            "in": "query",
+            "style": "form",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of requested page, indexed from 0. If null default value is assigned."
+            }
+          },
+          {
+            "name": "pageCount",
+            "in": "query",
+            "style": "form",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of items to return. If null default value is assigned."
+            }
+          }
+        ],
         "responses": {
           "200": {
             "description": "Success",
             "content": {
               "application/json": {
                 "schema": {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/components/schemas/ApplicationDTO"
-                  }
+                  "$ref": "#/components/schemas/GetApplicationsResponse"
                 }
               }
             }
           },
           "400": {
-            "description": "Client error"
+            "description": "Bad request - incorrect role."
```

Comment:
Imo "incorrect role" would look better in the 401 response, and 400 shold stay as sth more general like "client error"

Reply:
"Client error" is just default description of swashbuckle for generating swagger so it doesn't state anything, it is used everywhere because codes are usually self-descriptive. In this case I added decription by hand (when the pagination parameters were mandatory) and then deleted the part saying that and left this fragment. In our head it is explicitly returned when the token itself is correct, but the role doesn't match any of our defined roles. Can change it for 401 but I think that can help with diagnosing errors. Would love an opinion from every team on that.
@Lorak294
@Meeso1  
@pavonism 
@kubaurban 
@Sivvak

Reply:
IMO we should change it to 401

Reply:
I would opt for 401 with "Unauthorized - incorrect role." description, or just "Incorrect role". If we decide to change it to 401 then Bad request sounds misleading.